app/Http/Controllers/DashboardController.php

```diff
@@ -1,19 +1,19 @@
 <?php
 
 namespace App\Http\Controllers;
-
 use Illuminate\Http\Request;
 
 class DashboardController extends Controller
 {
 
     /**
-     * Show the application home page.
+     * Show the application dashboard page.
      *
      * @return \Illuminate\Contracts\Support\Renderable
      */
-    public function index()
+    public function index(Request $request)
     {
-        return view('dashboard');
+        $jobs = $request->user()->jobs()->get();
```

Comment:
I'm getting lost here, I can't find your User model, perhaps you are referring to Employer?? which in fact has the jobs relationship... 
in any case if that is what you seek here you can just call the relation as 
```
$employer->jobs; 
```
and that will load the data as well